packages/typescriptlang-org/src/components/search/ResultRow.tsx

```diff
@@ -0,0 +1,73 @@
+import { withPrefix } from "gatsby"
+import * as React from "react"
+
+import { cx } from "../../lib/cx"
+import { SearchHit } from "./types"
+
+import "./ResultRow.scss"
+
+export type ResultRowprops = {
+  exactMatch?: boolean
+  installer: [string, string]
+  hit: SearchHit
+}
+
+export const ResultRow: React.FC<ResultRowprops> = ({
+  exactMatch,
+  installer,
+  hit: {
+    description,
+    downloadsLast30Days,
+    humanDownloadsLast30Days,
+    modified,
+    name,
+    types,
+  },
+}) => {
+  const [icon, label] =
+    types.ts === "included"
+      ? ["in", "included"]
+      : ["dt", "from DefinitelyTyped"]
+
+  return (
+    <tr className={cx("resultRow", exactMatch && "resultRowExactMatch")}>
+      <td
+        className={cx(
+          "downloads",
+          downloadsLast30Days > 1_000_000
+            ? "downloadsMillion"
+            : downloadsLast30Days > 100_000 && "downloadsHundredThousand"
+        )}
+      >
+        {humanDownloadsLast30Days}
+      </td>
+      <td className="via">
+        <div
+          aria-label={`Types ${label}`}
+          className={cx("typeImage", icon)}
+          role="img"
+        />
+      </td>
+      <td className="name">
+        <strong>{name}</strong>
+        {description}
+      </td>
+      <td className="updated">{modified}</td>
+      <td className="install">
```

Comment:
Question: how would you like this to look on mobile? I'm guessing... hide the installation instructions, and on very small screens hide the "last updated"?

Reply:
Yep, I think just seeing the name and type and making the entire thing like to npm/DT is probably the right pattern - I'd imagine you're only doing this to prove that a dep has types in some format on mobile rather than doing anything with some depth